### PR TITLE
Add semicolons

### PR DESCRIPTION
**Pull request by kailevy**

### CODE REVIEW

**Comment by phuston** [comment]
Inappropriate, Kai.

File: routes/routes.js [hunk 1]
```diff
@@ -80,16 +80,16 @@ module.exports = function(app, passport) {
...
+	    // if user is authenticated in the session, carry on
 	    if (req.isAuthenticated())
 	        return next();
 
 	    // if they aren't redirect them to the home page
 	    res.redirect('/');
 	}
-}
\ No newline at end of file
+};
```
Comment (runnersaw) [inline routes/routes.js:95]: :+1:

**Comment by runnersaw** [comment]
:-1: but i'll merge anyway

File: routes/routes.js (hunk 1 above)
Comment (cwallac) [inline routes/routes.js]: https://www.youtube.com/watch?v=M94ii6MVilw

**Comment by cwallac** [comment]
https://www.youtube.com/watch?v=M94ii6MVilw
Sums up this PR

**Pull request merged**